scss/titles.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  align-items: center;
+  color: #000;
+  display: flex;
+  font-size: 1rem;
+  padding: 10px;  
+
+  &::before {
+    background: url('assets/triangle.svg') no-repeat right;
+    content: '';
+    margin-right: -10px;
+    max-width: 50px;
+    min-height: 50px;
```

Comment:
- title-decoration-height: 90px;
- title-decoration-width: 70px;

Check the project Setup card